sirepo/reply.py

```diff
@@ -133,6 +147,7 @@ def gen_exception(self, exc):
         # to the server, which will have code to handle this case.
         try:
             if isinstance(exc, sirepo.util.ReplyExc):
+                self.qcall.cookie.has_sentinel()
```

Comment:
Not used?

Reply:
[26d97f5](https://github.com/radiasoft/sirepo/pull/6751/commits/26d97f545074f52d3d357a8a3e70521ac279fccf)